### PR TITLE
Added npm files whitelist

### PR DESCRIPTION
**Pull request by TrySound**
For smaller package. Now it contains extra `coverage` and `nyc_output` folders.
500kb -> 100kb

### CODE REVIEW

**Comment by TrySound** [comment]
Make patch release please.

File: package.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
   "name": "async",
   "description": "Higher-order functions and common patterns for asynchronous code",
   "main": "lib/async.js",
+  "files": [
+    "lib"
+  ],
```
Comment (megawac) [inline package.json]: Agreed overall, but does npm support folders in `files` or do you specify exact path>?

Comment (TrySound) [inline package.json]: Yes. It works fine. I often do this thing.

**Comment by TrySound** [comment]
What is the problem with tests?

**Comment by megawac** [comment]
Looks good:

```
CHANGELOG.md LICENSE      README.md    lib          package.json

node_modules/async//lib:
async.js
```

**Comment by TrySound** [comment]
@megawac Cool! Thanks!

**Comment by aearly** [comment]
Nice catch -- this was  my mistake.

**Pull request merged**